.github/workflows/build.yml

```
@@ -33,18 +45,18 @@ jobs:
           # test group. It should be set to the highest test_group used in the
           # enos-run-matrices.
           MATRIX_MAX_TEST_GROUPS: 5
+          VAULT_VERSION: ${{ needs.set-product-version.outputs.vault-version }}
         run: |
-          echo "build-date=$(make ci-get-date)" >> $GITHUB_OUTPUT
-          echo "go-version=$(cat ./.go-version)" >> $GITHUB_OUTPUT
-          echo "matrix-test-group=$(make ci-get-matrix-group-id)" >> $GITHUB_OUTPUT
+          echo "build-date=$(make crt-get-date)" >> $GITHUB_OUTPUT
```

```suggestion
          echo "build-date=$(make ci-get-date)" >> $GITHUB_OUTPUT
```

I found this by diffing the oss and ent branches.  Also I don't see a crt-get-date target in the makefile on this branch.